includes/server-timing/object-cache.copy.php

```diff
@@ -58,17 +58,17 @@ function perflab_load_server_timing_api_from_dropin() {
 
 		$plugins_dir = defined( 'WP_PLUGIN_DIR' ) ? WP_PLUGIN_DIR : WP_CONTENT_DIR . '/plugins';
 		$plugin_dir  = $plugins_dir . '/performance-lab/';
-		if ( ! file_exists( $plugin_dir . 'server-timing/load.php' ) ) {
+		if ( ! file_exists( $plugin_dir . 'includes/server-timing/load.php' ) ) {
 			$plugin_dir = $plugins_dir . '/performance/';
-			if ( ! file_exists( $plugin_dir . 'server-timing/load.php' ) ) {
+			if ( ! file_exists( $plugin_dir . 'includes/server-timing/load.php' ) ) {
 				return;
 			}
 		}
 
-		require_once $plugin_dir . 'server-timing/class-perflab-server-timing-metric.php';
-		require_once $plugin_dir . 'server-timing/class-perflab-server-timing.php';
-		require_once $plugin_dir . 'server-timing/load.php';
-		require_once $plugin_dir . 'server-timing/defaults.php';
+		require_once $plugin_dir . 'includes/server-timing/class-perflab-server-timing-metric.php';
+		require_once $plugin_dir . 'includes/server-timing/class-perflab-server-timing.php';
+		require_once $plugin_dir . 'includes/server-timing/load.php';
+		require_once $plugin_dir . 'includes/server-timing/defaults.php';
```

Comment:
@thelovekesh Now that I see this change, moving the files for the Server-Timing API is tricky from a backward compatibility perspective. Here's why:
* The `wp-content/object-cache.php` drop-in may already be placed on some sites, from an older Performance Lab version. As such, it would try to load the old paths, which wouldn't work. It wouldn't cause a fatal error because of the checks above, but it just wouldn't load the API.
* In order to fix that, we'll have to enhance the logic that places that drop-in file (i.e. copies from `object-cache.copy.php` to the actual drop-in) so that it detects whether the current file is outdated.
* I'm happy to open another PR for that. However, at a minimum, let's update the drop-in version number in this PR from `2` to `3` (in both the file header and constant).

Reply:
@thelovekesh I've opened #1047 for the purpose of updating an older version of the drop-in, please take a look. Depending which of the two PRs gets merged first, the other one will require a minor update (to sync the version numbers).